lang/en/local_linkchecker_robot.php

```diff
@@ -81,7 +81,7 @@
 $string['size'] = 'Size';
 $string['url'] = 'URL';
 $string['mimetype'] = 'Mime type';
-$string['broken'] = 'Broken URL';
+$string['brokenlinksandurls'] = 'Broken URL';
```

Comment:
I've fixed this by reverting back all code references to as they were originally: get_string('broken'...). Then mapped this 'broken' string to 'Broken links / URLs' in language file.